packages/dcos-integration-test/extra/test_service_discovery.py

```diff
@@ -2,20 +2,22 @@
 import logging
 
 import socket
+from typing import Any
 
 import pytest
 import requests
 import retrying
 import test_helpers
 from dcos_test_utils import marathon
+from dcos_test_utils.dcos_api import DcosApiSession
 
 __maintainer__ = 'urbanserj'
 __contact__ = 'dcos-networking@mesosphere.io'
 
 DNS_ENTRY_UPDATE_TIMEOUT = 60  # in seconds
 
 
-def _service_discovery_test(dcos_api_session, docker_network_bridge):
+def _service_discovery_test(dcos_api_session: DcosApiSession, docker_network_bridge: Any) -> None:
```

Comment:
What type is `docker_network_bridge`?

Reply:
it has `bool` data types, will fix this.